src/obb/impl/main.cljs

```diff
@@ -23,19 +23,31 @@
   [x]
   (js/ObjectSpecifier.hasInstance x))
 
+(defn not-object-specifier-pred [f]
+  (fn [x]
+    (when-not (object-specifier? x)
+      (f x))))
+
+(set! interop/invoke-instance-method impl.sci/invoke-instance-method)
+
+(set! interop/invoke-static-method impl.sci/invoke-static-method)
+
+(set! clojure/map? (not-object-specifier-pred map?))
+
+(set! clojure/meta (not-object-specifier-pred meta))
```

Comment:
This is the real fix. I change with-redefs to set! since we want to make this a permanent change and not have the bindings restored after the eval, which wouldn't work if the eval was async.